src/tye/InitHost.cs

```diff
@@ -77,7 +88,7 @@ public static (string, string) CreateTyeFileContent(FileInfo? path, bool force)
                     }
 
                     // If the input file is a sln/project then place the config next to it
-                    outputFilePath = Path.Combine(directory.FullName, "tye.yaml");
+                    outputFilePath = Path.Combine(directory.FullName, outputFilePath);
```

Comment:
What does this do with our comments? https://github.com/dotnet/tye/pull/355/files#diff-03e018baedc83b3a7dd33c431001ccfdR48 when you do JSON?

Reply:
Comments are not supported in JSON.  Also consider the following note from http://www.yaml.org/spec/1.2/spec.html section 3.2.3.3. Comments

> Comments are a presentation detail and must not have any effect on the serialization tree or representation graph...